src/commands/deployment/get.ts

```diff
@@ -30,6 +30,8 @@ export enum OUTPUT_FORMAT {
   JSON = 2
 }
 
+const timeInterval = 5000;
```

Comment:
can we move this into `watchGetDeployments`'s body or add it as an option for the function? don't like having floating top level variables unless multiple things need to use it.

Reply:
If @samiyaakhtar wants to keep the variable on top level scope, can we add some doc around it to say what the variable does?

Reply:
I'm fine with keeping it in the body of the function if you don't prefer having floating top level const variables, i thought we wanted constants that change functionality to be more visible

Reply:
I'll revert it for now. I think we can later decide on how to structure the code. I was talking to Samiya about how it would also be nice to have constant strings like error messages etc. in their own file. It helps with consistency and reusing them etc. It doesn't seem like a priority for now though. It's just something to think about.

Reply:
Sounds good, sorry about the confusion here @edaena